### PR TITLE
Add ToEqualOneOf

### PR DESCRIPTION
**Pull request by samsonasik**
in favor of #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,10 +24,18 @@ require_once 'vendor/kahlan/extra-matcher/kahlan-config.php';
 
 ## Documentation
 
-**toBeOneOf($expected)**
+**toBeOneOf($expected)** // identic comparation
```
Comment (jails) [inline README.md]: strict comparisons ?

File: README.md [hunk 2]
```diff
@@ -24,10 +24,18 @@ require_once 'vendor/kahlan/extra-matcher/kahlan-config.php';
...
 
 ```php
 it("passes if $actual is present in $expected", function() {
     expect(3)->toBeOneOf([1, 2, 3]);
 });
 ```
+
+**toEqualOneOf($expected)** // non-identic comparation
```
Comment (jails) [inline README.md]: loose comparisons ?

**Comment by jails** [comment]
Comment aside, looks good to me :+1: !

File: README.md (hunk 1 above)
Comment (samsonasik) [reply README.md]: updated.

File: README.md (hunk 2 above)
Comment (samsonasik) [reply README.md]: updated.

**Pull request merged**